### PR TITLE
Winner Auction

### PR DESCRIPTION
**Pull request by hzhu1990**

### CODE REVIEW

File: app/src/main/java/io/github/keibai/models/Auction.java [hunk 1]
```diff
@@ -73,4 +73,12 @@ public String toString() {
                 ", id=" + id +
                 '}';
     }
+
+    public String getWinnerMessage() {
+        return "You have won in Auction " + this.id;
```
Comment (zurfyx) [inline app/src/main/java/io/github/keibai/models/Auction.java]: Wouldn't this fit best in the activity? A model should avoid returning humanized data such as sentences.

File: app/src/main/java/io/github/keibai/models/Auction.java [hunk 2]
```diff
@@ -73,4 +73,12 @@ public String toString() {
...
+    }
+
+    public String getStartTime() {
+        return this.startTime.toString();
```
Comment (zurfyx) [inline app/src/main/java/io/github/keibai/models/Auction.java]: Same thing here, the model should store the Date but the toString part should better be done by each of the views which are using it.

File: app/src/main/java/io/github/keibai/models/Auction.java (hunk 1 above)
Comment (hzhu1990) [reply app/src/main/java/io/github/keibai/models/Auction.java]: I think it doesn't matter, besides it shows me options where I can create the method, and one of the option is in Auction.

Comment (hzhu1990) [reply app/src/main/java/io/github/keibai/models/Auction.java]: besides I need the Auction id, which is easier to obtain if I am in the Auction.java

**Review by etorres404 — COMMENTED** [review]
Start time of auction model should be retrieved using the public field, instead of the getter.

**Review by etorres404 — APPROVED** [review]

**Pull request merged**